cfpb/design-manual
# Update to use family-linked webfonts and stop using mixins

Switches from using our old `u-webfont-` mixins to simple CSS styling, in accordance with the current version of Capital Framework which defaults to family-linked webfonts.

## Changes

- All `u-webfont-` mixins converted to simple CSS styling

## Removals

- `@webfont-` variables that duplicated the ones now in CF.

## Testing

1. Pull branch
1. `grunt`
1. `bundle exec jekyll serve watch`
1. Visit http://localhost:4000/ and browse around; see fonts looking as they should

[Preview this PR without the whitespace changes](?w=0)

## Checklist
